test/e2e/suite_test.go

```diff
@@ -251,7 +251,7 @@ func convertToPowershellorCmdCommandIfNecessary(command string) string {
 	case "while true; do echo $(date -u) >> /mnt/test-1/data; sleep 3600; done":
 		return "while (1) { Add-Content -Encoding Ascii C:\\mnt\\test-1\\data.txt $(Get-Date -Format u); sleep 3600 }"
 	case "echo 'hello world' >> /mnt/test-1/data && while true; do sleep 3600; done":
-		return "echo hello world>> C:\\mnt\\test-1\\data.txt&sleep 3600"
+		return "echo 'hello world' | Out-File -Append -FilePath C:\\mnt\\test-1\\data.txt; Get-Content C:\\mnt\\test-1\\data.txt | findstr 'hello world'; Start-Sleep 3600"
```

Comment:
what is `Get-Content C:\\mnt\\test-1\\data.txt | findstr 'hello world'` for? use `Out-File -Append` fixed this issue, right?

Reply:
I doubt whether this works since there are still failures for test `should create a deployment object, write and read to it, delete the pod and write and read to it again`, will disable this test first.

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/97439/pull-kubernetes-e2e-azure-disk-windows/1341339223176777728

Reply:
Hmmm. It looks like a timeout error and not because of the content of the file created by the test. The fix worked when I ran the test in my test cluster and in the automated tests for this PR. I suspect there is another cause for this failure.

> Unexpected error:
        <*errors.errorString | 0xc000891d80>: {
            s: "error waiting for deployment \"azuredisk-volume-tester-5b76m\" status to match expectation: deployment status: v1.DeploymentStatus{ObservedGeneration:1, Replicas:1, UpdatedReplicas:1, ReadyReplicas:0, AvailableReplicas:0, UnavailableReplicas:1, Conditions:[]v1.DeploymentCondition{v1.DeploymentCondition{Type:\"Available\", Status:\"False\", LastUpdateTime:v1.Time{Time:time.Time{wall:0x0, ext:63744233412, loc:(*time.Location)(0x2f9e780)}}, LastTransitionTime:v1.Time{Time:time.Time{wall:0x0, ext:63744233412, loc:(*time.Location)(0x2f9e780)}}, Reason:\"MinimumReplicasUnavailable\", Message:\"Deployment does not have minimum availability.\"}, v1.DeploymentCondition{Type:\"Progressing\", Status:\"True\", LastUpdateTime:v1.Time{Time:time.Time{wall:0x0, ext:63744233412, loc:(*time.Location)(0x2f9e780)}}, LastTransitionTime:v1.Time{Time:time.Time{wall:0x0, ext:63744233412, loc:(*time.Location)(0x2f9e780)}}, Reason:\"ReplicaSetUpdated\", Message:\"ReplicaSet \\\"azuredisk-volume-tester-5b76m-5f865796b8\\\" is progressing.\"}}, CollisionCount:(*int32)(nil)}",
        }

Reply:
looks like this commit fixed the issue: https://github.com/kubernetes-sigs/azuredisk-csi-driver/commit/f2c303b9a92395af675b61c88b49884f19c74418
I will continue monitoring the windows e2e test, it's been flacky for a very long time.

Reply:
Ah, thanks for catching that! :) I missed setting `useCMD` to `false` in my original PR. :(